packages/common/docs/Palette.md

```diff
@@ -0,0 +1,357 @@
+    .columns(["Category", "Value"])
+    .data([
+        ["Year 1", 34],
+        ["Year 5", 34],
+        ["Year 6", 55],
+        ["Year 4", 89]
+    ])
+    .render()
+    ;
+```
+
+**Note:**  "Year 5" and "Year 6" now have the same colors as "Year 2" and "Year 3" from the first example above.
+
+##  Ordinal
+
+Ordinal palettes are typically used in visualizations with discrete values.  They include the following defaults:
+
+```sample
+import { OrdinalSample } from "./palette/ordinal.js";
+
+const os = new OrdinalSample()
+    .target("target")
+    .render()
+    ;
+```
+
+### Palette.ordinal
+
+<!--meta:Palette.ordinal
+{
+    "id": 19908,
+    "name": "ordinal",
+    "kind": 32,
+    "kindString": "Variable",
+    "flags": {
+        "isExported": true,
+        "isConst": true
+    },
+    "sources": [
+        {
+            "fileName": "Palette.ts",
+            "line": 318,
+            "character": 20
+        }
+    ],
+    "type": {
+        "type": "reference",
+        "name": "fetchOrdinalItem",
+        "id": 19873
+    },
+    "defaultValue": " fetchOrdinalItem",
+    "folder": "packages/common",
+    "signatures": [
+        {
+            "id": 19874,
+            "name": "fetchOrdinalItem",
+            "kind": 4096,
+            "kindString": "Call signature",
+            "flags": {},
+            "type": {
+                "type": "array",
+                "elementType": {
+                    "type": "intrinsic",
+                    "name": "string"
+                }
+            }
+        },
+        {
+            "id": 19875,
+            "name": "fetchOrdinalItem",
+            "kind": 4096,
+            "kindString": "Call signature",
+            "flags": {},
+            "parameters": [
+                {
+                    "id": 19876,
+                    "name": "id",
+                    "kind": 32768,
+                    "kindString": "Parameter",
+                    "flags": {},
+                    "type": {
+                        "type": "intrinsic",
+                        "name": "string"
+                    }
+                },
+                {
+                    "id": 19877,
+                    "name": "colors",
+                    "kind": 32768,
+                    "kindString": "Parameter",
+                    "flags": {
+                        "isOptional": true
+                    },
+                    "type": {
+                        "type": "array",
+                        "elementType": {
+                            "type": "intrinsic",
+                            "name": "string"
+                        }
+                    }
+                }
+            ],
+            "type": {
+                "type": "reference",
+                "name": "ordinalPalleteFunc",
+                "id": 19869
+            }
+        }
+    ]
+}
+-->
+
+* When **id** and **colors** are both omitted: Returns a string array containing all the known ordinal palette IDs.
```

Comment:
Rather than saying `When **id** and **colors** are both omitted:`, this would be more straight forward `Palette.ordinal.fetchOrdinalItem():`

Reply:
I have tweaked the docs to make clearer